index.html

```diff
@@ -30,7 +34,31 @@ <h2 id="logo">Sajjad</h2>
         </li>
       </ul>
     </nav>
-  </header>
+    </div>
+    </header> -->
+    <header>
+      <nav class="navbar">
+        <a href="#" class="nav-branding">Sajjad</a>
+
+        <ul class="nav-menu">
+          <li class="nav-item">
+            <a href="#" class="nav-link">Portfolio</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Contact</a>
+          </li>
+        </ul>
+        
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
```

Comment:
- [ ] Kindly add the backdrop blur filter effect to the modal to match the Figma design.
- [ ] Please make sure that when the user clicks on a nav link (either in the desktop navbar or the mobile menu), the menu should close on the mobile menu and the page is scrolled to the appropriate section of the page. You can use `href` properties to do so.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/168102495-4802c42f-155a-4c9d-ad85-e9a7be652db9.png) | ![image](https://user-images.githubusercontent.com/62290545/168102564-79de9207-ae94-4aab-ae6a-02a921f8fb90.png) |